lib/mix/triplex.ex

```diff
@@ -22,7 +22,7 @@ defmodule Mix.Triplex do
   @doc """
   Returns the path for your tenant migrations.
   """
-  def migrations_path(repo \\ config().repo)
+  #def migrations_path(repo \\ config().repo)
```

Comment:
Rollback this comment.

```suggestion
  def migrations_path(repo \\ config().repo)
```